chained/tlsfrag.go

```diff
@@ -0,0 +1,53 @@
+package chained
+
+import (
+	"net"
+	"strconv"
+	"strings"
+
+	"github.com/Jigsaw-Code/outline-sdk/transport/tlsfrag"
+
+	"github.com/getlantern/common/config"
+)
+
+func tlsFragConn(conn net.Conn, pc *config.ProxyConfig) net.Conn {
+	fragFunc, ok := makeFragFunc(pc)
+	if ok {
+		if _, ok := conn.(*net.TCPConn); !ok {
+			return conn
+		}
+		tlsFragConn, err := tlsfrag.WrapConnFunc(conn.(*net.TCPConn), fragFunc)
+		if err != nil {
+			return conn
+		}
+		return tlsFragConn
+	}
+	return conn
+}
+
+func makeFragFunc(pc *config.ProxyConfig) (tlsfrag.FragFunc, bool) {
+	fragStr, ok := pc.PluggableTransportSettings["tlsfrag"]
+	if !ok {
+		return nil, false
+	}
+
+	// fragStr should be of the form <frag func> or <frag func>:<func config>.
```

Comment:
Does this comment make sense to you @myleshorton?  I wasn't sure what would be the most straight-forward in terms of nomenclature or formatting.

Reply:
I added a line below this with an example